soapui/src/main/java/com/eviware/soapui/support/editor/EditorView.java

```diff
@@ -59,5 +59,5 @@
 
     public void requestFocus();
 
-    boolean supportsContentType(String contentType );
+    int getSupportScoreForContentType(String contentType );

```

Comment:
This is a new method in the base interface. I am not sure that this is safe and will not break backward compatibility (e.g. for plugins). May be will more safe to add a default implementation for this method.

Reply:
I suggest to mark supportContentType with @deprecated and add new getSupportScoreForContentType with default implementation

Reply:
or default implementation could be kind of 
```
boolean supportsContentType (String contentType) {
  return getSupportScoreForContentType(contentType) > 1;
}
```

Reply:
please add also Java docs

Reply:
supportContentType has been added recently in Ole's initial implementation: https://github.com/SmartBear/soapui/pull/213. But I will add Javadocs.

Reply:
fixed